config/routes.rb

```diff
@@ -155,6 +155,21 @@
         get '/delete/:id' => 'other_qualifications/destroy#confirm_destroy', as: :confirm_destroy_other_qualification
         delete '/delete/:id' => 'other_qualifications/destroy#destroy'
       end
+
+      scope '/referees' do
+        get '/' => 'referees#index', as: :referees
+        get '/new' => 'referees#new', as: :new_referee
+        post '/' => 'referees#create', as: :create_referee
```

Comment:
Nit: you can reuse the `candidate_interface_referees_path` helper instead of creating a new one here (since it's the same URL).